### DISCUSSION TITLE
When's the next update gonna be released?

### DISCUSSION THREAD

**Discussion by ReportCardsMC** [Romitou/MongoSK#11] [Q&A] [answered]
Out of curiosity, when is this next update gonna be released? I've seen some more pushes for features I'd really like, if possible is there an estimated time? If not that's fine as well

**Comment by Romitou** [comment, marked as answer]
Hello, I initially thought of releasing a new release once I had a few more new features, but since some features take time to develop, I'm thinking of making smaller releases instead. So I will release a new release tonight!

In any case, don't forget that you can download MongoSK with the latest changes in [Actions](https://github.com/Romitou/MongoSK/actions) where you can try the latest additions.

Cheers!
